tests/unit/headless_browser_test.ts

```diff
@@ -182,6 +182,28 @@ Rhum.testPlan("tests/unit/headless_browser_test.ts", () => {
     );
   });
 
+  Rhum.testSuite("evaluatePage()", () => {
```

Comment:
Note to self too: we'll need to add error handling - what if a user types `evaluatePage("vfkjgjgrbj")`

Reply:
if we want `evaluatePage` to act like the console, then `evaluatePage("vfkjgjgrbj")` will return `"vfkjgjgrbj"` . it is a string expression... BUT, we should add tests that throws for things like `evaluatePage(aaa)`

Reply:
I believe that in other API methods, they do`this.checkForError(result)`, ill give it a try and see what happens

Reply:
Result inside the evaluate page:
```
{
  result: {
    type: "object",
    subtype: "error",
    className: "ReferenceError",
    description: "ReferenceError: fvejfvkjfkj is not defined\n    at <anonymous>:1:1",
    objectId: '{"injectedScriptId":2,"id":1}'
  },
  exceptionDetails: {
    exceptionId: 1,
    text: "Uncaught",
    lineNumber: 0,
    columnNumber: 0,
    scriptId: "12",
    exception: {
      type: "object",
      subtype: "error",
      className: "ReferenceError",
      description: "ReferenceError: fvejfvkjfkj is not defined\n    at <anonymous>:1:1",
      objectId: '{"injectedScriptId":2,"id":2}'
    }
  }
}
```

Reply:
so like if someone did add invalid input,we'd usually get:

![image](https://user-images.githubusercontent.com/47337480/104854305-cdf09b80-58fd-11eb-89de-b99654899b55.png)